### DISCUSSION TITLE
2023.11.20 第十四讲 世界变化快

### DISCUSSION THREAD

**Discussion by Lewisrxliu** [RoadSideUniversity/ClubHouse2024#37] [General]
世界只争朝夕 我们如何相处

**Comment by Lewisrxliu** [comment]
世界的终极是人
OpenAI 被刷屏的不合理， 戏剧化的变化也许是顺理成章的？ 开源世界 与 商业世界 的必经之路？

https://wallstreetcn.com/articles/3702329
https://www.zhihu.com/question/630622928

**Comment by Lewisrxliu** [comment]
Artist Min：NFT虚拟世界对年轻艺术家的不友好， 独立艺术创作者的未来之路在大机构精英圈重资本的压迫下空间狭窄。市场化成本过高没有成功范式。感谢 XP参与BOT项目的体验分享：收藏,策展,画廊,投资,艺术家共同体，需要区块链底层与市场推广AIGC共同构建新种子新土壤。Shawn: web2 & web3 的区别，新生态系统可以加入的验证，IP protection, scalable. HengSir:在爬到山顶时的底线思维，如果NFT在 web3无法突破是因为技术底层还没有构建完成, Too: 厌倦是因为没有找到爱，“从前车马很慢，书信很远，一生只够爱一个人”-木心《从前慢》

**Comment by Lewisrxliu** [comment]
世界变化快， XP: 东亚文化对不确定性不够尊重， 真相不一定是真相， 感觉和表达都是值得怀疑的， 逻辑和推理比较关键。
过程和结果都很重要。Too: 我们都是在对话中， 可能需要关注对方的情绪，随意的对话， 跑题带来的未知可能会带来新知。
感谢Bing的跑题： 体育运动一席谈

**Reply by too** [reply]
感谢书记员！我顺便补充一点，对话中关注的不止是情绪吧，还包括对话对方的其他了解，和一定的互动，可能更关键一点是不能顾着自己说，而要注意及时获取反馈

**Reply by Lewisrxliu** [reply]
是的是的！感谢补充，I like it

**Reply by Lewisrxliu** [reply]
极限运动爱好者帮助挑战人类理性的边际（XP)，感谢人类中的冒险因子

**Reply by Lewisrxliu** [reply]
https://en.wikipedia.org/wiki/Lanfang_Republic

**Reply by Lewisrxliu** [reply]
感谢bing的分享
https://zh.m.wikipedia.org/zh-hans/红溪惨案
